jipdate.py

```diff
@@ -498,6 +498,20 @@ def get_header():
 
     return ("\n".join(yml_iter) + "\n\n") if yml_iter is not None else "\n"
 
+def get_editor():
+    global g_yml_config
+
+    try:
+        yml_iter = g_yml_config['text-editor']
+    except:
+        # Probably no "text-editor" section in the yml-file.
+        return True
+
+    if yml_iter == "no" or yml_iter == "NO" or yml_iter == "No":
```

Comment:
Looks like i have never really tested it as -p was always in my command line :-(

-p arg is still useful because it also removes the push to the server.